guides/integrations-api/general-concepts/api-versioning.md

```diff
@@ -0,0 +1,95 @@
+# New API versioning explained
+
+## Overview
+
+Starting with Shopware version 6.4.0.0, we decided to change our API versioning strategy. 
+This article will cover what has been done and changed, how it used to be and how the version strategy looks like now.
+
+### Versioning prior to 6.3.5.0
```

Comment:
Starting with Shopware 6.4.0.0, we decided [...]

Versioning prior to 6.3.5.0 

So what now ? Did we change this in 6.4.0.0 or 6.3.5.0 ?